src/Components/Status/Status.js

```diff
@@ -23,7 +23,7 @@ useEffect(() => {
   return (
     <div className="status-container">
       <p className="status-header">welcome back!</p>
-      <p className="vendor-name">{currentVendor.name}!!!</p>
```

Comment:
not the exclamation points:( Just kidding all the code looks good so far and should interfere with the what we did hopefully!